src/core/mesh/qgsmeshlayerlabelprovider.h

```diff
@@ -0,0 +1,117 @@
+/***************************************************************************
+                         qgsmeshlayerlabelprovider.h
+                         ---------------------
+    begin                : November 2023
+    copyright            : (C) 2023 by LutraConsulting
+    email                : info at lutraconsulting dot co dot uk
```

Comment:
it would be better to have actual author who started the code (here and in other new files) - either Alex or yourself...

Reply:
I've put @alexbruy's details there, he was the orchestrator!